cg/services/pacbio/metrics/metrics_parser.py

```diff
@@ -1,72 +1,37 @@
 from pathlib import Path
-from typing import Any, Type
-
-from cg.constants.constants import FileFormat
-from cg.constants.pacbio import PacBioDirsAndFiles
-from cg.io.controller import ReadFile
 from cg.services.pacbio.metrics.models import (
-    BaseMetrics,
     ControlMetrics,
     HiFiMetrics,
+    PacBioMetrics,
     PolymeraseMetrics,
     ProductivityMetrics,
     SmrtlinkDatasetsMetrics,
 )
-from cg.utils.files import get_file_in_directory
+from cg.services.pacbio.metrics.utils import (
+    parse_control_metrics,
+    parse_hifi_metrics,
+    parse_polymerase_metrics,
+    parse_productivity_metrics,
+    parse_dataset_metrics,
+)
 
 
-class MetricsParser:
+class PacBioMetricsParser:
```

Comment:
Anyone going into this module does not what to be overwhelmed by details and lots of helper methods in the class. It is better to separate them. It is also easier to follow the control flow if you don't store calculated paths on the class.